src/main/java/lotto/controller/LottoController.java

```diff
@@ -15,21 +20,25 @@
 
     private static final String REGEX = ", ";
 
-    private LottoTickets lottoTickets;
-    private WinningLotto winningLotto;
-    private Payment payment;
-
-    public LottoController() {}
+    public LottoController() {
+    }
 
     public void run() {
-        payment = new Payment(ParseUtils.parseInt(InputView.getInputMoney()));
-        lottoTickets = new LottoTickets(payment.count());
-        showLottoTickets();
-        winningLotto = createWinningLotto();
-        OutputView.printResultMessage(lottoTickets.getResult(winningLotto), payment.getPayment());
+        final Payment payment = new Payment(ParseUtils.parseInt(InputView.getInputMoney()));
+        final SelfLottoCount selfLottoCount = new SelfLottoCount(payment.count(),
+            ParseUtils.parseInt(InputView.getBuySelfLottoCount()));
+        final List<List<Integer>> selfLottoTickets = buySelfLottoTickets(selfLottoCount);
```

Comment:
수동으로 산 것도 Lotto이기때문에 아래처럼 넘길수는 없을까요? 
```suggestion
        final List<Lotto> selfLottoTickets = buySelfLottoTickets(selfLottoCount);
```

Reply:
좀 더 공부해보겠습니다!